modules/rotating-certificate-authority

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// security/certificate_types_descriptions/etcd-certificates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rotating-certificate-authority_{context}"]
+=== Rotating the etcd certificate authority
```

Comment:
looks good as a whole! we need to also mention the `metrics-signer-ca` rotation, which is equally important for metrics and alerting